e2e_playwright/st_altair_chart_basic_select_test.py

```diff
@@ -129,7 +129,7 @@ def test_interval_bar_chart_displays_selection_text(app: Page):
 
     expected_prefix = "Bar chart with selection_interval:"
     expected_selection = re.compile(
-        "\\{'selection': \\{'param_1': \\{'a': \\['A', 'B'\\], 'b': \\[44.\\d+, 4(5|6).\\d+\\]\\}\\}\\}"
+        "\\{'selection': \\{'param_1': \\{'a': \\['A', 'B'\\], 'b': \\[.+, .+\\]\\}\\}\\}"
```

Comment:
We don't need to get down to the numbers here, only checking that there is a valid selection is fine. Otherwise, this will require a lot of updates whenever small things change.

Reply:
Fine with this change, but is this relevant to the change itself (min height of most elements)?

Reply:
The test fails otherwise. The reason is that there is one button in the test script which now has a slightly different height. This causes the selected area to be slightly different for all the charts underneath the button